website/content/partials/pki-forwarding-bug.mdx

```diff
@@ -0,0 +1,10 @@
+## PKI Certificate Generation Forwarding Regression
+
+A bug introduced in Vault 1.8 causes certificates issue by the PKI secrets engine made on a performance
+secondary node to be forwarded to a Vault cluster's primary node.  This affects horizontal scalability, but
```

Comment:
I don't expect the horizontal scalability is the real issue most users, it's more about the certs/CRLs being in the wrong place.  If we do want to retain this bit, I might expand it a little - not everyone's going to follow what we mean by horizontal scalability.

Reply:
Okay, I'll just drop that part.